<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RiotApiLoginScreen</class>
 <widget class="QDialog" name="RiotApiLoginScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login Screen</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>591</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>371</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Riot API Token:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>371</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Summoner Name:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="serverComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>79</width>
     <height>26</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>RU</string>
   </property>
   <item>
    <property name="text">
     <string>RU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EUW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EUNE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Server:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RiotApiLoginScreen</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RiotApiLoginScreen</receiver>
   <slot>verifyInputAndOpenMainWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>verifyInputAndOpenMainWindow()</slot>
 </slots>
</ui>
